src/main/java/duke/Task.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+public abstract class Task {
```

Comment:
I think its abit weird to call this an abstract class when theres no abstract method.
Maybe my understanding is wrong haha, I'm sorry in advance